### ISSUE TITLE
Exception while accessing project build

### ISSUE THREAD

**Issue by thc202** [crowdin/crowdin-api-client-java#25]
Using version 1.1.2.

Steps to reproduce the issue:
1. Trigger a project build:
```Java
Long projectId = ...; // Valid project ID.
Credentials credentials = ...; // Valid credentials.
new Client(credentials).getTranslationsApi().buildProjectTranslation(projectId, new BuildProjectTranslationRequest());
```
(Let me know if you need the project ID used.)
2. Note that it fails with an exception:
```
HttpException(error=HttpException.Error(code=null, message=Cannot deserialize instance of `java.util.ArrayList` out of START_OBJECT token
 at [Source: (String)"{"data":{"id":1,"projectId":1234,"status":"inProgress","progress":0,"attributes":{"branchId":null,"targetLanguageIds":[],"exportTranslatedOnly":false,"skipUntranslatedFiles":false,"exportApprovedOnly":false}}}"; line: 1, column: 82] (through reference chain: com.crowdin.client.translations.model.ProjectBuildResponseObject["data"]->com.crowdin.client.translations.model.ProjectBuild["attributes"])))
```
3. Attempting to use `checkBuildStatus` or `listProjectBuilds` will also result in a similar exception.

I'm not sure if this is a client issue or a server one, the API definition says that those API endpoints return a project build with a list of attributes but the server is currently just returning a single attribute (i.e. an object instead of a list). Is the client/API definition that's incorrect or the server should be really returning a list?

**Comment by andrii-bodnar** [comment]
Hi, @thc202!
Just checked this and it seems like a mistake in API definition and it should be a single attribute object.

**Comment by andrii-bodnar** [comment]
Fixed in Release [1.1.3](https://github.com/crowdin/crowdin-api-client-java/releases/tag/1.1.3)

**Comment by thc202** [comment]
Thanks!
